_data/sites.json

```diff
@@ -4806,6 +4806,17 @@
         ]
     },
 
+    {
+        "name": "Hotspot Shield",
+        "url": "",
```

Comment:
Nice investigation. If there's no URL to delete, just use the base URL of the service.